secrets_manager/README.md

```diff
@@ -0,0 +1,51 @@
+# Secrets Manager
+
+Module to create secrets in secrets manager
+
+## Resources
+
+- Tags
```

Comment:
Please add the other resources deployed and required environment variables here. ie AWS_DEFAULT_REGION and AWS_PROFILE.  AWS_PROFILE is also missing in TESTING.md. Testing and module works fine, just need those to changes to docs

Reply:
add as part of prerequisites section before running as module